src/atopile/front_end.py

```diff
@@ -1137,8 +1204,38 @@ def visitFunctional(
         self, ctx: ap.FunctionalContext
     ) -> fab_param.ParameterOperatable:
         if ctx.name():
-            # TODO: implement min/max
-            raise NotImplementedError
+            name = self.visitName(ctx.name())
+            operands = [self.visitBound(b) for b in ctx.bound()]
+            if name == "min":
```

Comment:
We can probably come up with a better pattern for these functions. These are the only cases so far, but I could imagine expansion as we decide to expand the language